Build failure: mpv-unwrapped

### Steps To Reproduce

Steps to reproduce the behavior:
1. build mpv-unwrapped on x86_64-darwin

### Build log

```
this derivation will be built:
  /nix/store/f3ygg3lmj3vs4zj3jlwal2wqvm91bhca-mpv-0.38.0.drv
building '/nix/store/f3ygg3lmj3vs4zj3jlwal2wqvm91bhca-mpv-0.38.0.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/rr912g9f1i10vb7d60bbz4njzxn4zr89-source
source root is source
Running phase: patchPhase
applying patch /nix/store/95ppcqmb4xabr261d0i9hiwsbnfxlw29-0001-fix-darwin-build.patch
patching file osdep/mac/input_helper.swift
patching file audio/out/ao_avfoundation.m
/private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS /private/tmp/nix-build-mpv-0.38.0.drv-0/source
patching script interpreter paths in docutils-wrapper.py dylib_unhell.py file2string.py gen-interface-changes.py macos-sdk-version.py macos-swift-lib-directory.py matroska.py osxbundle.py stats-conv.py gen-osd-font.sh idet.sh
docutils-wrapper.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
dylib_unhell.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
file2string.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
gen-interface-changes.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
macos-sdk-version.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
macos-swift-lib-directory.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
matroska.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
osxbundle.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
stats-conv.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3"
gen-osd-font.sh: interpreter directive changed from "#!/bin/sh" to "/nix/store/8prdfb52i3sy1wf64w03apn481585i4f-bash-5.2p26/bin/sh"
idet.sh: interpreter directive changed from "#!/bin/sh" to "/nix/store/8prdfb52i3sy1wf64w03apn481585i4f-bash-5.2p26/bin/sh"
/private/tmp/nix-build-mpv-0.38.0.drv-0/source
Running phase: configurePhase
mesonConfigurePhase flags: --prefix=/nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0 --libdir=/nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/lib --libexecdir=/nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/libexec --bindir=/nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/bin --sbindir=/nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/sbin --includedir=/nix/store/sdvgqari9mf1mjb09xgyaj4bw5yrr8g3-mpv-0.38.0-dev/include --mandir=/nix/store/l8q3yv3v0scya7lv1i2vpy4jnl148m1f-mpv-0.38.0-man/share/man --infodir=/nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/share/info --localedir=/nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/share/locale -Dauto_features=auto -Dwrap_mode=nodownload --buildtype=plain -Ddefault_library=shared -Dlibmpv=true -Dlibarchive=enabled -Dmanpage-build=enabled -Dcdda=disabled -Ddvbin=disabled -Ddvdnav=disabled -Dopenal=enabled -Dsdl2=disabled -Dswift-build=enabled -Dmacos-cocoa-cb=enabled -Dvideotoolbox-pl=disabled
The Meson build system
Version: 1.4.0
Source dir: /private/tmp/nix-build-mpv-0.38.0.drv-0/source
Build dir: /private/tmp/nix-build-mpv-0.38.0.drv-0/source/build
Build type: native build
Project name: mpv
Project version: 0.38.0
C compiler for the host machine: clang (clang 15.0.7 "clang version 15.0.7")
C linker for the host machine: clang ld64 609
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program python3 found: YES (/nix/store/6jy28lfjr42adzi10p158aq6m1q9qgs0-python3-3.11.9/bin/python3)
Found pkg-config: YES (/nix/store/sblqjb14ckz1xrzr4an26ff70dx9v8cl-pkg-config-wrapper-0.29.2/bin/pkg-config) 0.29.2
Run-time dependency libavcodec found: YES 60.31.102
Run-time dependency libavfilter found: YES 9.12.100
Run-time dependency libavformat found: YES 60.16.100
Run-time dependency libavutil found: YES 58.29.100
Run-time dependency libswresample found: YES 4.12.100
Run-time dependency libswscale found: YES 7.5.100
Run-time dependency libplacebo found: YES 6.338.2
Run-time dependency libass found: YES 0.17.1
Compiler for C supports arguments -Werror=implicit-function-declaration: YES
Compiler for C supports arguments -Wno-missing-field-initializers: YES
Compiler for C supports arguments -Wno-sign-compare: YES
Compiler for C supports arguments -Wno-unused-parameter: YES
Compiler for C supports arguments -Wno-cast-function-type: YES
Compiler for C supports arguments -Wempty-body: YES
Compiler for C supports arguments -Wdisabled-optimization: YES
Compiler for C supports arguments -Wstrict-prototypes: YES
Compiler for C supports arguments -Wno-format-zero-length: YES
Compiler for C supports arguments -Wno-redundant-decls: YES
Compiler for C supports arguments -Wvla: YES
Compiler for C supports arguments -Wno-format-truncation: NO
Compiler for C supports arguments -Wimplicit-fallthrough: YES
Compiler for C supports arguments -fno-math-errno: YES
Compiler for C supports arguments -Wformat -Werror=format-security: YES
Compiler for C supports link arguments -Wl,-z,noexecstack: NO
Compiler for C supports link arguments -Wl,--nxcompat,--no-seh,--dynamicbase: NO
Run-time dependency dl found: YES
Library atomic found: NO
Compiler for C supports link arguments -rdynamic: YES
Run-time dependency threads found: YES
Header "pthread.h" has symbol "pthread_condattr_setclock" with dependency threads: NO
Run-time dependency appleframeworks found: YES (Cocoa, IOKit, QuartzCore)
Checking for function "fork" : YES
Checking for function "ppoll" : NO
Library windowsapp skipped: feature uwp disabled
Checking for function "glob" : YES
Header "sys/vt.h" has symbol "VT_GETMODE" : NO
Header "sys/consio.h" has symbol "VT_GETMODE" : NO
Checking for function "fstatfs" : YES
Checking for function "fstatfs" : NO
Checking if "vector check" compiles: YES
Program /private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/docutils-wrapper.py found: YES (/private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/docutils-wrapper.py)
Program /private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/file2string.py found: YES (/private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/file2string.py)
Program /private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/matroska.py found: YES (/private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/matroska.py)
Dependency libcdio skipped: feature cdda disabled
Dependency libcdio_paranoia skipped: feature cdda disabled
Has header "linux/dvb/frontend.h" skipped: feature dvbin disabled
Dependency dvdnav skipped: feature dvdnav disabled
Dependency dvdread skipped: feature dvdnav disabled
Run-time dependency iconv found: YES
Run-time dependency mujs found: YES 1.3.4
Run-time dependency lcms2 found: YES 2.16
Run-time dependency libarchive found: YES 3.7.4
Run-time dependency libavdevice found: YES 60.3.100
Run-time dependency libbluray found: YES 1.3.4
Library m found: YES
Library rt found: NO
Run-time dependency lua found: YES 5.2.4
Run-time dependency rubberband found: YES 3.3.0
Dependency sdl2 skipped: feature sdl2 disabled
Run-time dependency uchardet found: YES 0.0.8
Did not find CMake 'cmake'
Found CMake: NO
Run-time dependency vapoursynth found: NO (tried pkgconfig, framework and cmake)
Run-time dependency vapoursynth-script found: NO (tried pkgconfig, framework and cmake)
Run-time dependency zimg found: YES 3.0.5
Run-time dependency zlib found: YES 1.3.1
Run-time dependency alsa found: NO (tried pkgconfig, framework and cmake)
Run-time dependency appleframeworks found: YES (Foundation, AudioToolbox)
Header "AudioToolbox/AudioToolbox.h" has symbol "kAudioUnitSubType_RemoteIO" : NO
Run-time dependency appleframeworks found: YES (CoreMedia, AVFoundation)
Run-time dependency appleframeworks found: YES (CoreFoundation, CoreAudio, AudioUnit, AudioToolbox)
Run-time dependency jack found: NO (tried pkgconfig, framework and cmake)
Run-time dependency openal found: YES 1.23.1
Library OpenSLES found: NO
Header "sys/soundcard.h" has symbol "SNDCTL_DSP_HALT" : NO
Run-time dependency libpipewire-0.3 found: NO (tried pkgconfig, framework and cmake)
Run-time dependency libpulse found: NO (tried pkgconfig, framework and cmake)
Run-time dependency sndio found: NO (tried pkgconfig, framework and cmake)
Header "audioclient.h" has symbol "IAudioClient" skipped: feature wasapi disabled
Run-time dependency caca found: YES 0.99.beta20
Has header "d3d9.h" skipped: feature direct3d disabled
Run-time dependency libdrm found: NO (tried pkgconfig, framework and cmake)
Dependency gbm skipped: feature gbm disabled
Run-time dependency libjpeg found: YES 3.0.3
Dependency shaderc skipped: feature shaderc disabled
Run-time dependency libsixel found: NO (tried pkgconfig, framework and cmake)
Checking for function "shm_open" : YES
Dependency spirv-cross-c-shared skipped: feature spirv-cross disabled
Run-time dependency wayland-client found: NO (tried pkgconfig, framework and cmake)
Run-time dependency wayland-cursor found: NO (tried pkgconfig, framework and cmake)
Run-time dependency wayland-protocols found: NO (tried pkgconfig, framework and cmake)
Run-time dependency xkbcommon found: NO (tried pkgconfig, framework and cmake)
Run-time dependency wayland-client found: NO (tried pkgconfig, framework and cmake)
Has header "linux/input-event-codes.h" with dependency wayland-client: NO
Program wayland-scanner found: NO
Run-time dependency x11 found: YES 1.8.9
Run-time dependency xscrnsaver found: YES 1.2.4
Run-time dependency xext found: NO (tried pkgconfig, framework and cmake)
Run-time dependency xpresent found: NO (tried pkgconfig, framework and cmake)
Run-time dependency xrandr found: NO (tried pkgconfig, framework and cmake)
Dependency xv skipped: feature xv disabled
Run-time dependency appleframeworks found: YES (OpenGL)
Library EGL skipped: feature egl-angle-lib disabled
Run-time dependency egl found: NO (tried pkgconfig, framework and cmake)
Library EGL skipped: feature egl-android disabled
Dependency wayland-egl skipped: feature egl-wayland disabled
Run-time dependency vulkan found: NO (tried pkgconfig and system)
Checking for function "vkCreateDisplayPlaneSurfaceKHR" with dependency vulkan: NO
Run-time dependency ffnvcodec found: NO (tried pkgconfig, framework and cmake)
Header "OpenGLES/ES3/glext.h" has symbol "GL_RGB32F" : NO
Run-time dependency libva found: NO (tried pkgconfig, framework and cmake)
Dependency libva-drm skipped: feature vaapi-drm disabled
Dependency libva-wayland skipped: feature vaapi-wayland disabled
Dependency libva-x11 skipped: feature vaapi-x11 disabled
Dependency libva-win32 skipped: feature vaapi-win32 disabled
Dependency vdpau skipped: feature vdpau disabled
Dependency appleframeworks (modules: CoreVideo) skipped: feature videotoolbox-pl disabled
Program /private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/macos-sdk-version.py found: YES (/private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/macos-sdk-version.py)
Message: Detected macOS sdk path: /nix/store/smdg628nf328pggk107pdir34mky782k-SDKs/MacOSX11.0.sdk
Message: Detected macOS SDK: 11.0
meson.build:1505: WARNING: add_languages is missing native:, assuming languages are wanted for both host and build.
Objective-C compiler for the host machine: cc (clang 15.0.7)
Objective-C linker for the host machine: cc ld64 609
Program xcrun found: YES (/nix/store/5dwqj8z88ni78sym3w7hal0fxa53ch8h-xcrun/bin/xcrun)
Program /nix/store/qazkmdyj9qym8wzy36lpgjcdaagr146v-swift-wrapper-5.8/bin/swift found: YES (/nix/store/qazkmdyj9qym8wzy36lpgjcdaagr146v-swift-wrapper-5.8/bin/swift)
Message: Detected Swift version: 5.8
Has header "AppKit/NSTouchBar.h" : YES
Program /private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/macos-swift-lib-directory.py found: YES (/private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/macos-swift-lib-directory.py)
Message: Detected Swift library directory: /nix/store/20d0sxwqc80pl65dgz6c589ng5dx9rm6-swift-5.8-lib/lib/swift/macosx
Program rst2man found: YES (/nix/store/yqn9icmrz7ljjfq789bf3v8f96lw9d9d-python3.11-docutils-0.21.2/bin/rst2man)
Program rst2html rst2html.py skipped: feature html-build disabled
Program rst2pdf skipped: feature pdf-build disabled
Configuring config.h using configuration
Message: List of enabled features: av-channel-layout avfoundation avif-muxer bsd-fstatfs build-date caca cocoa coreaudio cplugins darwin ffmpeg gl gl-cocoa glob glob-posix gpl iconv javascript jpeg jpegxl lavu-uuid lcms2 libarchive libass libavdevice libbluray libdl libplacebo lua mac-thread-name macos-cocoa-cb macos-media-player macos-touchbar openal posix posix-shm rubberband rubberband-3 swift uchardet vector videotoolbox-gl zimg zimg-st428 zlib
Fetching value of define "MPV_CLIENT_API_VERSION" : (((2) << 16) | (3) | 0UL)
Compiler for C supports link arguments -Wl,-Bsymbolic: NO
Program /private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/osxbundle.py found: YES (/private/tmp/nix-build-mpv-0.38.0.drv-0/source/TOOLS/osxbundle.py)
Build targets in project: 26

mpv 0.38.0

    d3d11          : NO
    javascript     : YES
    libmpv         : YES
    lua            : YES
    opengl         : YES
    vulkan         : NO
    wayland        : NO
    x11            : NO

  User defined options
    auto_features  : auto
    bindir         : /nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/bin
    buildtype      : plain
    default_library: shared
    includedir     : /nix/store/sdvgqari9mf1mjb09xgyaj4bw5yrr8g3-mpv-0.38.0-dev/include
    infodir        : /nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/share/info
    libdir         : /nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/lib
    libexecdir     : /nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/libexec
    localedir      : /nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/share/locale
    mandir         : /nix/store/l8q3yv3v0scya7lv1i2vpy4jnl148m1f-mpv-0.38.0-man/share/man
    prefix         : /nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0
    sbindir        : /nix/store/iw19dbm12icvrv4j89n03rbp58hr7wj7-mpv-0.38.0/sbin
    wrap_mode      : nodownload
    cdda           : disabled
    dvbin          : disabled
    dvdnav         : disabled
    libarchive     : enabled
    libmpv         : true
    macos-cocoa-cb : enabled
    manpage-build  : enabled
    openal         : enabled
    sdl2           : disabled
    swift-build    : enabled
    videotoolbox-pl: disabled

Found ninja-1.11.1 at /nix/store/iqqrcj6kf157q1qym8gn6ypgn8pi4hld-ninja-1.11.1/bin/ninja
mesonConfigurePhase: enabled\ parallel\ building
configurePhase completed in 45 seconds
Running phase: buildPhase
build flags: -j12
[1/240] Generating common/version.h with a custom command
[2/240] Generating ebml_defs.inc with a custom command
[3/240] Generating ebml_types.h with a custom command
[4/240] Generating etc/mpv-icon-8bit-16x16.png with a custom command
[5/240] Generating etc/mpv-icon-8bit-32x32.png with a custom command
[6/240] Generating etc/builtin.conf with a custom command
[7/240] Generating player/lua/assdraw.lua with a custom command
[8/240] Generating player/lua/options.lua with a custom command
[9/240] Generating etc/mpv-icon-8bit-64x64.png with a custom command
[10/240] Generating player/lua/defaults.lua with a custom command
[11/240] Generating player/javascript/defaults.js with a custom command
[12/240] Generating etc/mpv-icon-8bit-128x128.png with a custom command
[13/240] Generating player/lua/ytdl_hook.lua with a custom command
[14/240] Generating etc/input.conf with a custom command
[15/240] Generating player/lua/osc.lua with a custom command
[16/240] Generating player/lua/stats.lua with a custom command
[17/240] Generating player/lua/input.lua with a custom command
[18/240] Generating player/lua/auto_profiles.lua with a custom command
[19/240] Generating sub/osd_font.otf with a custom command
[20/240] Generating player/lua/console.lua with a custom command
[21/240] Generating TOOLS/osxbundle/osxbundle with a custom command
[22/240] Compiling C object mpv.p/osdep_main-fn-mac.c.o
[23/240] Generating manpages with a custom command
[24/240] Generating osdep/mac/swift_targets with a custom command
FAILED: osdep/mac/swift.swiftmodule osdep/mac/swift.h osdep/mac/swift.o
/nix/store/qazkmdyj9qym8wzy36lpgjcdaagr146v-swift-wrapper-5.8/bin/swift -frontend -c -sdk /nix/store/smdg628nf328pggk107pdir34mky782k-SDKs/MacOSX11.0.sdk -enable-objc-interop -emit-objc-header -parse-as-library -O -D HAVE_MACOS_COCOA_CB -D HAVE_MACOS_TOUCHBAR -D HAVE_MACOS_MEDIA_PLAYER -module-name swift -emit-module-path osdep/mac/swift.swiftmodule -import-objc-header /private/tmp/nix-build-mpv-0.38.0.drv-0/source/osdep/mac/app_bridge_objc.h -emit-objc-header-path osdep/mac/swift.h -o osdep/mac/swift.o ../osdep/mac/application.swift ../osdep/mac/app_hub.swift ../osdep/mac/event_helper.swift ../osdep/mac/input_helper.swift ../osdep/mac/libmpv_helper.swift ../osdep/mac/log_helper.swift ../osdep/mac/menu_bar.swift ../osdep/mac/option_helper.swift ../osdep/mac/precise_timer.swift ../osdep/mac/presentation.swift ../osdep/mac/swift_compat.swift ../osdep/mac/swift_extensions.swift ../osdep/mac/type_helper.swift ../video/out/mac/common.swift ../video/out/mac/title_bar.swift ../video/out/mac/view.swift ../video/out/mac/window.swift ../video/out/cocoa_cb_common.swift ../video/out/mac/gl_layer.swift ../osdep/mac/remote_command_center.swift ../osdep/mac/touch_bar.swift -I. -I/private/tmp/nix-build-mpv-0.38.0.drv-0/source -I/nix/store/nf7diy25qf4469bnsnr7z6c8vqvnkxa4-libplacebo-6.338.2/include
../osdep/mac/libmpv_helper.swift:138:20: warning: 'CGLFlushDrawable' was deprecated in macOS 10.14: OpenGL API deprecated. (Define GL_SILENCE_DEPRECATION to silence these warnings)
        if !skip { CGLFlushDrawable(ctx) }
                   ^
../osdep/mac/log_helper.swift:23:18: error: 'Logger' is only available in macOS 11.0 or newer
    let logger = Logger(subsystem: "io.mpv", category: "mpv")
                 ^
../osdep/mac/log_helper.swift:21:7: note: add @available attribute to enclosing class
class LogHelper {
      ^
../osdep/mac/log_helper.swift:54:66: error: 'appendInterpolation(_:align:privacy:)' is only available in macOS 11.0 or newer
            logger.log(level: loggerMapping[type] ?? .default, "\(message, privacy: .public)")
                                                                 ^
../osdep/mac/log_helper.swift:54:66: note: add 'if #available' version check
            logger.log(level: loggerMapping[type] ?? .default, "\(message, privacy: .public)")
                                                                 ^
../osdep/mac/log_helper.swift:52:10: note: add @available attribute to enclosing instance method
    func send(message: String, type: Int) {
         ^
../osdep/mac/log_helper.swift:21:7: note: add @available attribute to enclosing class
class LogHelper {
      ^
../osdep/mac/log_helper.swift:54:64: error: 'OSLogMessage' is only available in macOS 11.0 or newer
            logger.log(level: loggerMapping[type] ?? .default, "\(message, privacy: .public)")
                                                               ^
../osdep/mac/log_helper.swift:54:64: note: add 'if #available' version check
            logger.log(level: loggerMapping[type] ?? .default, "\(message, privacy: .public)")
                                                               ^
../osdep/mac/log_helper.swift:52:10: note: add @available attribute to enclosing instance method
    func send(message: String, type: Int) {
         ^
../osdep/mac/log_helper.swift:21:7: note: add @available attribute to enclosing class
class LogHelper {
      ^
../video/out/mac/title_bar.swift:39:29: warning: 'light' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to an appropriate NSAppearance value.
            if material == .light || material == .dark || material == .mediumLight ||
                            ^
../video/out/mac/title_bar.swift:39:51: warning: 'dark' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to an appropriate NSAppearance value.
            if material == .light || material == .dark || material == .mediumLight ||
                                                  ^
../video/out/mac/title_bar.swift:39:72: warning: 'mediumLight' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to an appropriate NSAppearance value.
            if material == .light || material == .dark || material == .mediumLight ||
                                                                       ^
../video/out/mac/title_bar.swift:40:29: warning: 'ultraDark' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to an appropriate NSAppearance value.
               material == .ultraDark
                            ^
../video/out/mac/title_bar.swift:216:53: warning: 'dark' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to an appropriate NSAppearance value.
        case "14", "dark":                  return .dark
                                                    ^
../video/out/mac/title_bar.swift:217:53: warning: 'light' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to an appropriate NSAppearance value.
        case "15", "light":                 return .light
                                                    ^
../video/out/mac/title_bar.swift:218:53: warning: 'mediumLight' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to anappropriate NSAppearance value.
        case "16", "mediumLight":           return .mediumLight
                                                    ^
../video/out/mac/title_bar.swift:219:53: warning: 'ultraDark' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to an appropriate NSAppearance value.
        case "17", "ultraDark":             return .ultraDark
                                                    ^
../video/out/mac/view.swift:35:9: warning: 'wantsBestResolutionOpenGLSurface' was deprecated in macOS 10.14: Use NSOpenGLView instead.
        wantsBestResolutionOpenGLSurface = true
        ^
../video/out/mac/gl_layer.swift:70:16: warning: 'CAOpenGLLayer' was deprecated in macOS 10.14: OpenGL is deprecated
class GLLayer: CAOpenGLLayer {
               ^
../video/out/mac/gl_layer.swift:115:9: warning: 'CGLSetParameter' was deprecated in macOS 10.14: OpenGL API deprecated. (Define GL_SILENCE_DEPRECATION to silence these warnings)
        CGLSetParameter(cglContext, kCGLCPSwapInterval, &i)
        ^
../video/out/mac/gl_layer.swift:188:13: warning: 'NSDisableScreenUpdates()' was deprecated in macOS 10.11: As of 10.11 it is not generally necessary to take explicit action to achieve visual atomicity. +[NSAnimationContext runAnimationGroup:] and other similar methods can be used when a stronger than normal need for visual atomicity is required. The NSAnimationContext methods do not suffer from the same performance problems as NSDisableScreenUpdates.
            NSDisableScreenUpdates()
            ^
../video/out/mac/gl_layer.swift:195:13: warning: 'NSEnableScreenUpdates()' was deprecated in macOS 10.11: As of 10.11 it is not generally necessary to take explicit action to achieve visual atomicity. +[NSAnimationContext runAnimationGroup:] and other similar methods can be used when a stronger than normal need for visual atomicity is required. The NSAnimationContext methods do not suffer from the same performance problems as NSEnableScreenUpdates.
            NSEnableScreenUpdates()
            ^
../video/out/mac/gl_layer.swift:291:23: warning: 'CGLChoosePixelFormat' was deprecated in macOS 10.14: OpenGL API deprecated. (Define GL_SILENCE_DEPRECATION to silence these warnings)
                err = CGLChoosePixelFormat(format, &pix, &npix)
                      ^
../video/out/mac/gl_layer.swift:307:36: warning: 'CGLErrorString' was deprecated in macOS 10.14: OpenGL API deprecated. (Define GL_SILENCE_DEPRECATION to silence these warnings)
        let errS = String(cString: CGLErrorString(err))
                                   ^
../video/out/mac/gl_layer.swift:320:21: warning: 'CGLCreateContext' was deprecated in macOS 10.14: OpenGL API deprecated. (Define GL_SILENCE_DEPRECATION to silence these warnings)
        let error = CGLCreateContext(pixelFormat, nil, &context)
                    ^
../video/out/mac/gl_layer.swift:323:40: warning: 'CGLErrorString' was deprecated in macOS 10.14: OpenGL API deprecated. (Define GL_SILENCE_DEPRECATION to silence these warnings)
            let errS = String(cString: CGLErrorString(error))
                                       ^
ninja: build stopped: subcommand failed.
```

### Notify maintainers

@AndersonTorres @fpletz @globin @ma27 @tadeokondrak

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-darwin"`
 - host os: `Darwin 23.5.0, macOS 10.16`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.2`
 - channels(root): `""`
 - channels(user): `""`
 - nixpkgs: `/nix/store/5jgh89kgmrb687c254wxdac4cj5hqjw8-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/video/mpv/default.nix
+++ b/pkgs/applications/video/mpv/default.nix
@@ -121,10 +121,7 @@ let
 
   overrideSDK =
     platform: version:
-    platform
-    // lib.optionalAttrs (platform ? darwinMinVersion) {
-      darwinMinVersion = version;
-    };
+    platform // lib.optionalAttrs (platform ? darwinMinVersion) { darwinMinVersion = version; };
 
   stdenv' =
     if swiftSupport && stdenv.isDarwin && stdenv.isx86_64 then
@@ -357,9 +354,7 @@ stdenv'.mkDerivation (finalAttrs: {
     tests = {
       inherit (nixosTests) mpv;
 
-      version = testers.testVersion {
-        package = finalAttrs.finalPackage;
-      };
+      version = testers.testVersion { package = finalAttrs.finalPackage; };
       pkg-config = testers.hasPkgConfigModules {
         package = finalAttrs.finalPackage;
         moduleNames = [ "mpv" ];

--- a/pkgs/applications/video/mpv/default.nix
+++ b/pkgs/applications/video/mpv/default.nix
@@ -129,9 +129,9 @@ let
   stdenv' =
     if swiftSupport && stdenv.isDarwin && stdenv.isx86_64 then
       stdenv.override (old: {
-        buildPlatform = overrideSDK old.buildPlatform "10.15";
-        hostPlatform = overrideSDK old.hostPlatform "10.15";
-        targetPlatform = overrideSDK old.targetPlatform "10.15";
+        buildPlatform = overrideSDK old.buildPlatform "11.0";
+        hostPlatform = overrideSDK old.hostPlatform "11.0";
+        targetPlatform = overrideSDK old.targetPlatform "11.0";
       })
     else
       stdenv;

